playground/components/footer/footer.tsx

```diff
@@ -22,7 +22,7 @@ export const Info = styled('div')`
     margin: 0.25rem 0;
   }
 
-  @media screen and (min-width: 600px) {
+  @media screen and (min-width: 768px) {
```

Comment:
Reason for this change?

Reply:
Adding step count to the playground/examples there's a need to wrap the values earlier. Try resizing the window for yourself with(out) the change.

Reply:
And choosing `768` was convenient because it's a common breakpoint and the break is needed around `750` so simply couldn't resist (plus it adds a little margin to account for browser inconsistencies etc.).

Reply:
Do you want me to show you side-by-side screenshots?